onoff.js

```diff
@@ -261,5 +266,8 @@ class Gpio {
   }
 }
 
-exports.Gpio = Gpio;
+Gpio.HIGH = HIGH;
+Gpio.LOW = LOW;
 
+exports.Gpio = Gpio;
+module.exports = { Gpio };
```

Comment:
`exports.Gpio = Gpio;` was already there. Are both `exports.Gpio = Gpio;` and the new line `module.exports = { Gpio };` needed?

Reply:
the `module.exports` vs `exports` can be ignored...

as for `Gpio` vs `{ Gpio }`
the latter allows for multiple exports (example `{ Gpio, HIGH, LOW })` and in the requiring code `const { Gpio, HIGH, LOW} = require('onoff')`

both are "religion" so i will follow your previous style.